omegatrade-app/backend/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "stock-trade",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@google-cloud/spanner": "^5.4.0",
+    "bcryptjs": "^2.4.3",
+    "body-parser": "^1.19.0",
+    "cors": "^2.8.5",
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "fake-stock-market-generator": "^1.0.3",
+    "jsonwebtoken": "^8.5.1",
+    "node-schedule": "^1.3.2",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "nodemon": "^2.0.7"
+  }
+}
```

Comment:
I can see many unused dependencies here. I assume it will be used later?

Reply:
yes, 
1. fake-stock-market-generator
2. node-schedule
the above packages are necessary and going to be used in upcoming PRs